### ISSUE TITLE
internal errors exit with code 0

### ISSUE THREAD

**Issue by DetachHead** [PyCQA/pylint#5570]
### Bug description

when an [internal error](https://pycodequ.al/docs/pylint-messages/Internal.html) appears, the exit code is 0

```py
# test.py
foo = 1  # pylint: disable=redefined-builtin
```

### Configuration

```ini
[MESSAGE CONTROL]
enable=
    useless-suppression
```

### Command used

```shell
pylint test.py
```

### Pylint output

```shell
************* Module Sandpit.test
Sandpit\test.py:170:0: I0021: Useless suppression of 'redefined-builtin' (useless-suppression)
```

### Expected behavior

non-zero exit code

### Pylint version

```shell
pylint 2.12.2
astroid 2.9.0
Python 3.10.0rc2 (tags/v3.10.0rc2:839d789, Sep  7 2021, 18:51:45) [MSC v.1929 64 bit (AMD64)]
```

**Comment by DanielNoord** [comment]
The `I` doesn't reflect an internal error but an information error. Note that the documentation you linked are not from our own repository, but from another project that uses `pylint`. They seem to misclassify these issues, perhaps you can raise an issue against their repository?

That said, you could make `pylint` return non-zero for information messages. We do so ourself in our `pre-commit` with `"-`-fail-on=I`.
